crates/pallets/starknet/src/tests/invoke_tx.rs

```diff
@@ -80,26 +97,50 @@ fn given_hardcoded_contract_run_invoke_tx_then_it_works() {
 
         assert_eq!(pending_hashes[0], tx_hash);
         let events: Vec<StarknetEvent> = Starknet::tx_events(tx_hash);
-        println!("evens: {events:?}");
-
-        assert!(events.into_iter().any(|e| e
-            == StarknetEvent {
-                from_address: Starknet::fee_token_addresses().eth_fee_token_address,
-                content: EventContent {
-                    keys: vec![EventKey(
-                        Felt252Wrapper::from(get_selector_from_name(TRANSFER_SELECTOR_NAME).unwrap()).into(),
-                    )],
-                    data: EventData(vec![
-                        StarkFelt::try_from(BLOCKIFIER_ACCOUNT_ADDRESS).unwrap(),
-                        StarkFelt::try_from("0xdead").unwrap(),
-                        StarkFelt::try_from("0xb932").unwrap(),
-                        StarkFelt::from(0u128),
-                    ]),
-                },
-            },));
+        println!("events: {:?}", events);
+
+        assert!(events.into_iter().any(|e| e == StarknetEvent {
+            from_address: Starknet::fee_token_addresses().eth_fee_token_address,
+            content: EventContent {
+                keys: vec![EventKey(
+                    Felt252Wrapper::from(get_selector_from_name(TRANSFER_SELECTOR_NAME).unwrap()).into()
+                )],
+                data: EventData(vec![
+                    StarkFelt::try_from(BLOCKIFIER_ACCOUNT_ADDRESS).unwrap(),
+                    StarkFelt::try_from("0xdead").unwrap(),
+                    StarkFelt::try_from("0xb932").unwrap(),
+                    StarkFelt::from(0u128),
+                ]),
+            },
+        }));
     });
 }
 
+fn get_invoke_dummy_v0(chain_id: u64, nonce: Nonce) -> InvokeTransaction {
+    let contract_address = ContractAddress(PatriciaKey(
+        StarkFelt::try_from("0x03e437FB56Bb213f5708Fcd6966502070e276c093ec271aA33433b89E21fd31f").unwrap(),
+    ));
+    let calldata = Calldata(Arc::new(vec![]));
+    let value = Fee(0);
+    let gas_limit = 1000;
```

Comment:
You can define them directly as you construct inveke_tx_v0, as you never reuse them